include/service_control_client.h

```diff
@@ -281,6 +281,17 @@ class ServiceControlClient {
       DoneCallback on_check_done, TransportCheckFunc check_transport) = 0;
 
   // An async quota call.
+  // on_quota_done is called with the quota request status after cached
+  // quota_response is returned in case of cache hit, otherwise called after
+  // quota_response is returned from the Controller service.
+  // When quota cache is enabled, quota_response can be mutated to contain a
+  // gRPC server error status in its allocate_errors field. This allows caching
+  // for fail-close errors. In other words, if quota cache is enabled, callers
+  // should check quota_response.allocate_errors for Controller service quota
+  // errors and gRPC errors (typically these errors are client-side errors
+  // corresponding to 4xx in HTTP).
+  //
+  // quota_response must be alive until on_quota_done is called.
```

Comment:
nit: suggest we use backticks ` to capture code vars

\`quota_response\`

Same for all variables above

Reply:
Thanks! Done.

Reply:
I notice the other comments in this file do not have backticks. I will update them in a separate PR following this.